Block/BlockRenderer.php

```diff
@@ -93,7 +93,7 @@ public function render(BlockContextInterface $blockContext, Response $response =
             $response = $this->addMetaInformation($response, $blockContext, $service);
         } catch (\Exception $exception) {
             if ($this->logger) {
-                $this->logger->critical(sprintf('[cms::renderBlock] block.id=%d - error while rendering block - %s', $block->getId(), $exception->getMessage()));
+                $this->logger->error(sprintf('[cms::renderBlock] block.id=%d - error while rendering block - %s', $block->getId(), $exception->getMessage()));
```

Comment:
Nice change! Could you also linebreak this, and add the exception object to the context argument of error ? Like this : 

```php
$this->logger->error(sprintf(
    '[cms::renderBlock] block.id=%d - error while rendering block - %s',
    $block->getId(),
    $exception->getMessage()
), compact('exception'));
```